### PR TITLE
Add py3 support for speech commands example

### PR DESCRIPTION
**Pull request by daemon**
* Add imports for xrange and cast iterable to list

### CODE REVIEW

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

File: tensorflow/examples/speech_commands/train.py [hunk 1]
```diff
@@ -79,6 +79,7 @@
 
 import input_data
 import models
+from six.moves import xrange  # pylint: disable=redefined-builtin
```
Comment (caisq) [inline tensorflow/examples/speech_commands/train.py]: Actually, can you move this to line 78 (after `import numpy as np`)? The ordering is: builtin modules, third-party modules, project-specific modules. `six` is a third-party module.

**Review by caisq — DISMISSED** [review]

**Comment by drpngx** [comment]
Transient windows failure. Jenkins, test this please.

**Comment by caisq** [comment]
I manually fixed the import order, @daemon 

@tensorflow-jenkins test this please

**Review by drpngx — APPROVED** [review]

**Comment by drpngx** [comment]
@caisq I think it's ready.

**Pull request merged**